src/common/certificate-authorities/system-ca.test.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import https from "https";
+import { DSTRootCAX3 } from "./is-cert-active";
+import assert from "assert";
+import { describeIf } from "../../test-utils/skippers";
+import { platform } from "process";
+import { getDiForUnitTesting } from "../../main/getDiForUnitTesting";
+import injectSystemCAsOnMacOSInjectable from "./inject-system-cas-on-macos.injectable";
+import injectSystemCAsOnWindowsInjectable from "./inject-system-cas-on-windows.injectable";
+
+describe("CA tests", () => {
```

Comment:
This integration test only covers some odd corner-cases, and not the happy-path of tested module.